portfolio/src/main/webapp/music.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+       
+        <title>My Favorite Songs</title>   
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="title">
+        <h1 style="background-color:DodgerBlue;">
+            Music I Like by Genre (Not in any order)</h1>
+        </div>
+            <div id="list">
```

Comment:
This is a new div and it's on the same level with the previous div, so you should unindent and align it to be on the same spacing as the previous closing div!